src/display/dom_utils.js

```diff
@@ -160,14 +160,20 @@ PDFJS.isExternalLinkTargetSet = isExternalLinkTargetSet;
  * @param {HTMLLinkElement} link - The link element.
  * @param {Object} params - An object with the properties:
  * @param {string} params.url - An absolute URL.
+ * @param {boolean} params.newWindow - (optional) The 'NewWindow' property of
```

Comment:
Yes, as I mentioned above that PR should land first. After that, I plan on changing this line to use `target` instead :-)